### PR TITLE
ADEN-7869 GDPR check for mobile-app

### PR DESCRIPTION
**Pull request by pnather**

### CODE REVIEW

File: src/utils.js [hunk 1]
```diff
@@ -37,3 +37,7 @@ export function getJSON(url) {
         req.send(null);
     });
 }
+
+export function urlContainsParam(url, param) {
+    return new URL(url).searchParams.get(param) === 'true';
```
Comment (dmnsobczak) [inline src/utils.js]: Is it only string `true`? What about Boolean true and/or `1`? Please verify that with @gaperlinski.

File: src/index.js [hunk 2]
```diff
@@ -114,6 +115,9 @@ export default function main(options) {
         },
         window.location,
     );
-    instance.render();
+
+    if (!urlContainsParam(window.location.href, 'mobile-app')) {
```
Comment (dmnsobczak) [inline src/index.js]: You expect to get some instance returned from function `main()`. Please check if this won't brake anything on app/mw/f2.

File: src/utils.js [hunk 3]
```diff
@@ -37,3 +37,7 @@ export function getJSON(url) {
         req.send(null);
     });
 }
+
+export function urlContainsParam(url, param) {
```
Comment (dmnsobczak) [inline src/utils.js]: I would rename it to `isParameterSet` and move here `window.location.href` to call it just `isParameterSet('mobile-app')`.

File: src/utils.js (hunk 1 above)
Comment (pnather) [reply src/utils.js]: they set always to 'true'

**Pull request merged**